src/rpc/server/scala/serv/src/omega-edit-grpc-server.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2021 Concurrent Technologies Corporation.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const path = require('path')
+const os = require('os')
+const child_process = require('child_process')
+
+function runServer(args) {
+  const scriptDir = path.join(__dirname, 'node_modules/omega-edit/bin')
```

Comment:
This script should be sitting right next to the OS-specific scripts.  We need to find the code that tells us where _this_ script is located, resolve that directory and we'll find everything else we need from that point of reference.